internal/kernel_neon.h

```diff
@@ -727,7 +985,8 @@ struct NEON_64bit_GEMM_Int8Operands_LhsNonzero : KernelBase {
         "beq " GEMMLOWP_LABEL_AFTER_LOOP_LAST16 "f\n"
 
         // General loop.
-        GEMMLOWP_LABEL_LOOP ":\n"
```

Comment:
Just the effect of running clang-format on this file. I know it is a silly change, but are we ready to decide to give up clang-formatting this file? That would probably entail disabling formatting checks that we have internally, or else submitting any change to this file would require ignoring presubmit warnings?